shared/Microsoft.Extensions.WebEncoders.Sources/Properties/EncoderResources.cs

```diff
@@ -19,6 +19,11 @@ internal static class EncoderResources
         /// </summary>
         internal static readonly string WebEncoders_MalformedInput = "Malformed input: {0} is an invalid input length.";
 
+        /// <summary>
+        /// Invalid input, that doesn't conform a base64 string.
+        /// </summary>
+        internal static readonly string WebEncoders_InvalidInput = "The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.";
```

Comment:
This class has a nice TODO

```c#
// TODO using a resx file. project.json, unfortunately, fails to embed resx files when there are also compile items
// in the contentFiles section. Revisit once we convert repos to MSBuild
```

Shall I fix this -- or in a different PR?

Reply:
Do a different PR if it will be a noisy change.